README.md

```diff
@@ -110,6 +110,27 @@ variable if it is not already configured, and
 
 ### Installing HarbourBridge
 
+#### HarbourBridge on gCloud
+You can directly run HarbourBridge from the gCloud CLI instead of building it from source. In order to start using HarbourBridge via Gcloud, the user can install the harbourbridge component of gcloud by executing the below command:
```

Comment:
Link this to the gcloud installation instructions.
Also mention this is currently possible only on Linux

Reply:
Done